### ISSUE TITLE
Feedback and Arch Linux Install Support

### ISSUE THREAD

**Issue by PR454D** [foss42/apidash#87]
**Feedback**
UI looks nice. the response mimetype view is also a good feature.
I'd like to add PKGBUILD to install `apidash` on arch linux based systems.

**Comment by animator** [comment]
Hi @PR454D Thank you for the feedback. We are currently working on the release.
Will update the issue as soon as it is published.
Thanks!

**Comment by animator** [comment]
Hi @PR454D, We will investigate how to prepare an Arch Linux Installer.
Meanwhile, this issue is open for anyone to work on and send across a PR>

**Comment by PR454D** [comment]
Hey @animator, I'm already working on a PKGBUILD. It's a way to add packages in the community repos. I'll submit a pull request soon.

**Comment by animator** [comment]
Thanks @PR454D

**Comment by animator** [comment]
Hi @PR454D Can you please take a look at this Arch Linux build and try it out.
https://aur.archlinux.org/packages/apidash-bin

Looking forward to your feedback!

**Comment by PR454D** [comment]
yes, works exactly as expected.. thanks. This issue can be closed.

**Comment by animator** [comment]
Glad to hear it!
Cheers 🎉
